frontend/src/utils/tsxUtils.tsx

```diff
@@ -330,7 +330,6 @@ interface StatusIndicatorProps {
 @observer
 export class StatusIndicator extends Component<StatusIndicatorProps> {
 
-    static readonly progressStyle: CSSProperties = { minWidth: '300px', lineHeight: 0 } as const;
     static readonly statusBarStyle: CSSProperties = { display: 'flex', fontFamily: '"Open Sans", sans-serif', fontWeight: 600, fontSize: '80%' } as const;
     static readonly progressTextStyle: CSSProperties = { marginLeft: 'auto', paddingLeft: '2em' } as const;
```

Comment:
[question] Shouldn't this be removed as part of the `Progress` refactor? We can just apply the styles to a `Text` or `Box` element or similar from Chakra, so that
```tsx
<div style={StatusIndicator.progressTextStyle}>{this.props.progressText}</div>
```
becomes
```tsx
<Text ml="auto" pl="2em">{StatusIndicator.progressTextStyle}</Text>
```

Reply:
You're right, I'll go through this and apply Chakra's Style Props to the whole PR (`progressStyle` , `progressTextStyle` and others). I'm a bit new to Chakra, but I get the point now.

Reply:
No worries, just making sure :) Happy to check again